src/main/java/com/atlassian/db/replica/internal/ConnectionParameters.java

```diff
@@ -47,6 +51,24 @@ public void initialize(Connection connection) throws SQLException {
         if (networkTimeout != null) {
             networkTimeout.configure(connection);
         }
+        initializeRuntimeParameters(connection);
+    }
+
+    private void initializeRuntimeParameters(Connection connection) throws SQLException {
+        if (runtimeParameterChanges.isEmpty()) {
+            return;
+        }
+        try (Statement statement = connection.createStatement()) {
+            for (String parameterChange : runtimeParameterChanges) {
+                statement.execute(parameterChange);
+            }
+        }
+    }
+
+    public void addRuntimeParameterConfiguration(
```

Comment:
for now `runtimeParameterChanges` are only `statements`, do you expect other parameters like this? if yes which?

Reply:
I think I don't understand the question. What does it mean `runtimeParameterChanges are only statements`?

Reply:
this is NIT.
In other words, why `addRuntimeParameterConfiguration` ? 
not `addStatemets`, what do you have in mind?

Reply:
`set` sets runtime parameters for the current connection. We add the configurations (in a form of SQL commands), not statements.